commands/Tools/contact.js

```diff
@@ -0,0 +1,33 @@
+/* Add in the channel ID where you want to recieve reports in your app.js file Komada Configurations.
+  new Komada({
+    prefix: blah,
+    ..more here,
+    supportChannel: 'someID',
+    }).login(token);
+*/
+exports.run = async (client, msg, [type, ...content]) => {
+  const types = {
+    bug: "Bug Report",
+    idea: "New Idea",
+    message: "New Message",
+  };
+  return client.config.supportChannel.send(`${types[type]} From: ${msg.author} in ${msg.guild}\n\n ${content.toString().replace(/,/g, " ")}`);
```

Comment:
As you said in the documentation you just write for this command, supportChannel is a **id**. You shall redo the previous step with `exports.init` + `exports.channel`.

Also, your method will replace legit commas into spaces. `...content` is an Array, use [Array.prototype.join](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join) instead.